explorer/src/components/MixNodes/index.ts

```diff
@@ -35,5 +37,6 @@ export function mixNodeResponseItemToMixnodeRowType(item: MixNodeResponseItem):
     host: item?.mix_node?.host || '',
     layer: item?.layer || '',
     profit_percentage: `${profitPercentage}%`,
+    avg_uptime: avgUptime,
```

Comment:
I think you could assign `item.avg_uptime` directly here instead of creating a variable and then assigning.